File: src/main/java/io/github/icusystem/icu_connect/api_icu/StatusResponse.java
package io.github.icusystem.icu_connect.api_icu;

import java.util.List;

public class StatusResponse {

    public String DeviceState;
    public int LastFaceUpdate;
    public int LastActionUpdate;
    public String SingleState;
    public List<FaceDetect> FaceInFrame;
    public List<Detection> Detections;

}

File: src/main/java/io/github/icusystem/icu_connect/api_icu/StatusResponse.kt
package io.github.icusystem.icu_connect.api_icu
data class StatusResponse(
    val DeviceState:String,
    val LastFaceUpdate:Int,
    var LastActionUpdate:Int,
    var SingleState:String,
    var FaceInFrame:List<FaceDetect>,
    val Detections:List<Detection>
    )


